faker_test.go

```diff
@@ -721,6 +728,15 @@ func TestLangWithLen(t *testing.T) {
 	if rusLen != someStructWithLenAndLangRUS {
 		t.Errorf("Got %d, but expected to be %d as a string len", rusLen, someStructWithLenAndLangRUS)
 	}
+
+	err = isStringLangCorrect(someStruct.ValueJPN, LangJPN)
+	if err != nil {
+		t.Error(err.Error())
+	}
+	jaLen := utfLen(someStruct.ValueJPN)
+	if jaLen != someStructWithLenAndLangJPN {
+		t.Errorf("Got %d, but expected to be %d as a string len", rusLen, someStructWithLenAndLangJPN)
+	}
```

Comment:
```suggestion
	
	jpnLen := utfLen(someStruct.ValueJPN)
	if jpnLen != someStructWithLenAndLangJPN {
		t.Errorf("Got %d, but expected to be %d as a string len", jpnLen, someStructWithLenAndLangJPN)
	}
```

Reply:
Suggest to change variable name, and use `jpnLen` to the error

Reply:
thx!
Fixed it.
https://github.com/bxcodec/faker/pull/145/commits/b1772f05a498134f43f024d4caca111c623d5432